articles/iot-edge/tutorial-nested-iot-edge.md

```diff
@@ -589,6 +628,21 @@ Notice that the image URI that we used for the simulated temperature sensor modu
 
 On the device details page for your lower layer IoT Edge device, you should now see the temperature sensor module listed along the system modules as **Specified in deployment**. It may take a few minutes for the device to receive its new deployment, request the container image, and start the module. Refresh the page until you see the temperature sensor module listed as **Reported by device**.
 
+## IotEdge check
+Iotedge check can be performed in nested hierarchy, even if the children doesn't have direct internet access.
+When running "iotedge check" from the lower layer the program will try to pull the image from the parent through port 443.
+In this How-to, we use port 8000, so we need to specify it:
+
+   ```bash
+   iotedge check --diagnostics-image-name <parent_device_fqdn_or_ip>:8000/azureiotedge-diagnostics:1.2.0-rc2
```

Comment:
They can't use $upstream:8000 here? That only works in deployments?

Reply:
yes, it doesn't work here.